### PR TITLE
Feat geode object class heritage

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: src/opengeodeweb_back/geode_functions.py [hunk 1]
```diff
@@ -13,43 +13,47 @@
 import werkzeug
 
 # Local application imports
-from .geode_objects import objects_list
+from .geode_objects import geode_objects_dict
```
Comment (BotellaA) [inline src/opengeodeweb_back/geode_functions.py:16]: Why this change? This implies breaking change which implies. Be careful about not adding too much work

File: src/opengeodeweb_back/geode_functions.py [hunk 2]
```diff
@@ -13,43 +13,47 @@
...
 
 
-def get_input_factory(geode_object: str):
-    return objects_list()[geode_object]["input_factory"]
+def geode_object_value(geode_object: str):
```
Comment (BotellaA) [inline src/opengeodeweb_back/geode_functions.py:19]: What is the goal of this? What is the added value?

File: src/opengeodeweb_back/geode_functions.py (hunk 1 above)
Comment (JulienChampagnol) [reply src/opengeodeweb_back/geode_functions.py:16]: The name was a little bit confusing, it said list but it was actually a dict.
And since the name is just used in the functions, this isn't supposed to introduce any breaking change

File: src/opengeodeweb_back/geode_functions.py (hunk 2 above)
Comment (JulienChampagnol) [reply src/opengeodeweb_back/geode_functions.py:19]: This is mainly for easier testing

**Review by BotellaA — APPROVED** [review]
